components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -261,7 +261,13 @@ data class NimbusAppInfo(
      *
      * Examples: "nightly", "beta", "release"
      */
-    val channel: String
+    val channel: String,
+    /**
+     * Some specific context that the SDK doesn't need to be explicitly aware of.
+     * this is used to allow targeting on fields that are plaform or app specific
+     * that other consumers of the SDK shouldn't consume
```

Comment:
By this point, this is app developer focused documentation.

I suggest the docs should be written for them: e.g. application derived attributes measured by the application, but useful for targeting of experiments. e.g. `mapOf("userType" to "casual", "isFirstTime": "true")`.